### DISCUSSION TITLE
Suggestion: new diseases that usually leads to outbreaks in humanitarian settings

### DISCUSSION THREAD

**Discussion by ruoranepi** [epiverse-trace/epiparameter#276] [Ideas]
Cholera
Hepatitis A/E
Meningitis
Bacterial dysentery
Diptheria

**Comment by joshwlambert** [comment]
Thank you for contributing. I have added the diseases to the issue (#237) that tracks which diseases/pathogens should be added to the database. We will try to search the literature for epidemiological parameters for these diseases and add them to the package as soon as possible.
